front/app/containers/MainHeader/index.tsx

```diff
@@ -94,9 +70,9 @@ const StyledRightFragment = styled(Fragment)`
 
 const MainHeader = () => {
   const containerRef = useRef<HTMLDivElement>(null);
-  const locale = useLocale();
 
-  const [showNavBar, setShowNavBar] = useState<boolean>(false);
+  const [showMobileStickyNav, setShowMobileStickyNav] =
+    useState<boolean>(false);
```

Comment:
Not sure if this is the best name, but thought that "showNavBar" was not specific enough.